actors/actor-ot/src/receiver.rs

```diff
@@ -39,58 +46,63 @@ pub struct KOSReceiverActor<T, U> {
     pending_buffer: HashMap<String, oneshot::Sender<Result<Kos15IOReceiver<RandSetup>, OTError>>>,
 }
 
-impl<T, U> KOSReceiverActor<T, U>
-where
-    T: Channel<OTMessage, Error = std::io::Error> + Send + 'static,
-    U: MuxChannelControl<OTMessage> + Send + 'static,
-{
+impl KOSReceiverActor {
+    /// Creates a new KOSReceiverActor
+    ///
+    /// # Arguments
+    ///
+    /// * `config` - The configuration for the receiver
+    /// * `addr` - The address of the receiver
+    /// * `spawner` - The spawner to spawn the internal tasks
+    /// * `channel` - The channel over which OT splits are synchronized with the remote
+    ///               KOSSenderActor
+    /// * `mux_control` - The muxer which sets up channels with the remote KOSSenderActor
     pub fn new(
```

Comment:
Should we restrict this to `pub(crate)`, since we have the `create_` functions in `setup.rs`?